pkg/api/v1/resource/helpers.go

```diff
@@ -83,6 +83,8 @@ func PodRequests(pod *v1.Pod, opts PodResourcesOptions) v1.ResourceList {
 		addResourceList(reqs, containerReqs)
 	}
 
+	restartableInitContainerReqs := v1.ResourceList{}
+	initContainerReqs := v1.ResourceList{}
 	// init containers define the minimum of any resource
 	// Note: In-place resize is not allowed for InitContainers, so no need to check for ResizeStatus value
```

Comment:
Can we add a comment here to explain the logic of the resource calculation for sidecar containers? I would copy the relevent section from the KEP

Reply:
Can we add a comment here to explain the logic of the resource calculation for sidecar containers? Or link to the section in the KEP - https://github.com/kubernetes/enhancements/tree/master/keps/sig-node/753-sidecar-containers#resources-calculation-for-scheduling-and-pod-admission

Reply:
done.